### PR TITLE
Add storybook

### PR DESCRIPTION
**Pull request by gustaveen**
## What
Adding storybook setup

## Why
To enable smoother local development workflow and in the end an ui library of all our components

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "@segment/snippet": "^4.4.0",
     "@segment/utm-params": "^2.0.0",
     "@sentry/node": "^4.3.0",
+    "@types/storybook-react-router": "^1.0.1",
```
Comment (palmenhq) [inline package.json]: Imo `@types` should go in `devDependencies` :)

**Review by palmenhq — COMMENTED** [review]
Boom! Amaze!

File: package.json (hunk 1 above)
Comment (gustaveen) [reply package.json]: Ah of course, my bad!

**Review by palmenhq — APPROVED** [review]
✅✅✅

**Pull request merged**